src/components/seances/SeanceCard.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div class="seance-card">
+    <div class="seance-card__poster">
+      <img
+        :src="
+          $store.getters.movieList.filter((movie) => movie.id == movieId)[0]
```

Comment:
I suggest to also extract this to the computed value

Reply:
I wan't sure if this would work as computed since I need to pass a movieId value to this function so I made it a method? What do you think?

Reply:
no need for param: `movieId` is a prop so you have access to it's accessible under `this.movieId` 😉